ingestion_tools/scripts/importers/base_importer.py

```diff
@@ -301,12 +302,14 @@ class BaseFileImporter(BaseImporter):
     This class extends the BaseImporter to handle the import of files. This is suited for case where no transformations
     are required, and source files can be copied to the destination.
     """
+    def get_destination_path(self) -> str:
```

Comment:
just a reminder that we probably want to update the rest of our importers to use this when we get a chance!